lambdas/tests/unit/common/utils/request-utils.test.ts

```diff
@@ -0,0 +1,45 @@
+import { getClientIpAddress, getSessionId } from "../../../../src/common/utils/request-utils";
+
+describe("getClientIpAddress", () => {
```

Comment:
Could we wrap this in a `request-utils` block, similar to [session-service-test](https://github.com/alphagov/di-ipv-cri-common-lambdas/blob/main/lambdas/tests/unit/services/session-service.test.ts#L22)